zipline/data/_minute_bar_internal.pyx

```diff
@@ -100,17 +95,15 @@ def find_last_traded_position_internal(
     """
     cdef Py_ssize_t minute_pos, current_minute
 
-    minute_pos = int_min(
-        find_position_of_minute(market_opens, end_minute, minutes_per_day),
+    start_pos = searchsorted(minutes, start_minute, side='left')
```

Comment:
you should `cdef` `start_pos` and `end_pos` above, right? (and get rid of `current_minute`)